session_srtcp.go

```diff
@@ -112,28 +112,50 @@ func (s *SessionSRTCP) decrypt(buf []byte) error {
 		return err
 	}
 
+	forward := func(pkt rtcp.Packet) (closed bool, err error) {
+		for _, ssrc := range pkt.DestinationSSRC() {
+			r, isNew := s.session.getOrCreateReadStream(ssrc, s, newReadStreamSRTCP)
+			if r == nil {
+				return true, nil // Session has been closed
+			} else if isNew {
+				s.session.newStream <- r // Notify AcceptStream
+			}
+
+			readStream, ok := r.(*ReadStreamSRTCP)
+			if !ok {
+				return false, fmt.Errorf("failed to get/create ReadStreamSRTP")
+			}
+
+			_, err = readStream.write(decrypted)
```

Comment:
Confused why this is `decrypted`.

If a compound packet with `n` subpackets arrives won't this just deliver the compound packet `n` times?

Reply:
you're right, and your other comment below is related. good catch! let me take another pass.